python/cudf/cudf/core/reshape.py

```diff
@@ -978,7 +982,10 @@ def pivot(data, index=None, columns=None, values=None):
     if len(columns_index) != len(columns_index.drop_duplicates()):
         raise ValueError("Duplicate index-column pairs found. Cannot reshape.")
 
-    return _pivot(values, index, columns)
+    result = _pivot(values, index, columns)
+    if not values_is_list:
+        result.columns = result.columns.droplevel(0)
```

Comment:
If possible, we should do this by operating on the `ColumnAccessor` (`result._data`), rather  than using the `.columns` attribute (which is slow).

Reply:
Done.